recipes/bioconductor-dexseq/pythonpath.patch

```diff
@@ -0,0 +1,16 @@
+--- inst/python_scripts/dexseq_count.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_count.py.new	2017-06-16 14:14:38.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
++
+ import sys, itertools, optparse, warnings
+ 
+ optParser = optparse.OptionParser( 
+--- inst/python_scripts/dexseq_prepare_annotation.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_prepare_annotation.py.new	2017-06-16 14:14:55.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
As this is not a rule now please feel free to leave it like that. We should probably discuss this in an issue and find evidence how conda-forge is doing it.
Thanks for all your efforts!